### PR TITLE
add path for client_secrets.json in sample_tools.py

### PR DESCRIPTION
**Pull request by t-Asai**
I add path for client_secrets.json in sample_tools.py, because we have some accounts then we need to prepare some client_secrets.json.
To use the same script with several json files, I made each directory and set env files in the directories.

### CODE REVIEW

**Comment by t-Asai** [comment]
I signed it!

File: googleapiclient/sample_tools.py [hunk 1]
```diff
@@ -72,7 +72,7 @@ def init(argv, name, version, doc, filename, scope=None, parents=[], discovery_f
   # application, including client_id and client_secret, which are found
   # on the API Access tab on the Google APIs
   # Console <http://code.google.com/apis/console>.
-  client_secrets = os.path.join(os.path.dirname(filename),
+  client_secrets = os.path.join(os.path.dirname(filename), os.getcwd(),
```
Comment (jonparrott) [inline googleapiclient/sample_tools.py]: I don't think this works the way you expect. What's the case that this is supposed to fix?

**Comment by sduskis** [comment]
This PR is stale, and needs attention.  @t-Asai, do you want to follow up on comments, or should we close this?

**Comment by busunkim96** [comment]
Closing this as the original author @t-Asai has not responded. Please feel free to re-open.

**Pull request closed without merging**